common/src/main/java/io/seata/common/util/StringUtils.java

```diff
@@ -196,4 +197,32 @@ public static String toString(Object obj){
         }
         return sb.toString();
     }
+
+    /**
+     * Equals boolean.
+     *
+     * @param a the a
+     * @param b the b
+     * @return boolean
+     */
+    public static boolean equals(String a, String b) {
```

Comment:
What if the string is a blank character?should it be considered?

Reply:
like a="    "

Reply:
Do you mean  " "  and "" ?   I think there are not equals, maybe add new method `equalsIngoreBlank`

Reply:
yes

Reply:
> Do you mean " " and "" ? I think there are not equals, maybe add new method `equalsIngoreBlank`

can you add it in this PR,please?

Reply:
I will add it when we need it.

Reply:
> I will add it when we need it.

sure